runtime/onert/core/include/ir/train/ITrainableOperation.h

```diff
@@ -40,7 +40,21 @@ class ITrainableOperation : virtual public IOperation
   virtual void accept(OperationVisitor &v) const override = 0;
   virtual void accept(TrainableOperationVisitor &v) const = 0;
   virtual bool hasTrainableParameter() const = 0;
-  // TODO Add virtual methods related to training
+
+  // Update of the node will be disabled during traning
+  virtual void disableWeightsUpdate() = 0;
+  // Note that the update is disabled by default
+  // Update of the node will be enabled during traning
+  virtual void enableWeightsUpdate() = 0;
+  // Check if the node is trainable
+  virtual bool isTrainable() const = 0;
```

Comment:
(Optional)
How about the name to be changed to `isToBeTrained` or another name if this function is introduced check whether `TrainableOperation` is needed to be trained? The name `Trainable` is duplicated because the class name already contains `Trainable`.

Reply:
The motivation behind it was to match the other frameworks like `.trainble` field in Python, but I see your point. What's about `isWeightsUpdateEnabled` to match setters?
/cc @jyoungyun

Reply:
It looks good to me :)